controllers/pod_models.go

```diff
@@ -1250,6 +1256,39 @@ func NormalizeClusterSpec(spec *fdbtypes.FoundationDBClusterSpec, options Deprec
 	return nil
 }
 
+// GetPodDisruptionBudgets generates the disruption budgets that we need to
+// configure for a cluster.
+func GetPodDisruptionBudgets(cluster *fdbtypes.FoundationDBCluster) ([]policyv1beta1.PodDisruptionBudget, error) {
+	if !*cluster.Spec.EnablePodDisruptionBudget {
+		return nil, nil
+	}
+	budgets := make([]policyv1beta1.PodDisruptionBudget, 0, len(fdbtypes.ProcessClasses))
```

Comment:
so, this makes one PDB per process class. So each process class can be interrupted independently ? But we'll also get a PDB for Proxy and Resolver and so on, even when none are in use. So this is saying that FDB can handle 
1 storage + 1 transaction + 1 stateless + 1 resolution + 1 unset + 1 log ... all down at once.

I don't think that that is correct, is it?

Reply:
What advice would we give on the forums for someone planning manual migrations outside of kubernetes. Would it be 'move one process at a time', or 'no more than one stateful and one stateless at a time, and never more than $coordinatorcount/2 -1 a a time' ? or something like that : thats what I think we should aim to match.